tests/benchmark.py

```diff
@@ -300,104 +336,172 @@ def benchmark_medium_complex_object():
         test_object.append({str(random.random() * 20): int(random.random() * 1000000)})
-    run_encode(COUNT)
+    run_encode(COUNT, libraries)
 
     decode_data = json.dumps(test_object)
     test_object = None
-    run_decode(COUNT)
+    run_decode(COUNT, libraries)
 
     decode_data = None
 
 
-def benchmark_dict_of_arrays_of_dict_string_int_pairs():
+def benchmark_dict_of_arrays_of_dict_string_int_pairs(libraries, factor=1):
     global decode_data, test_object
     results_new_benchmark("Dict with 256 arrays with 256 dict{string, int} pairs")
-    COUNT = 50
+    COUNT = max(int(50 * factor), 1)
 
     test_object = {}
     for y in range(256):
         arrays = []
         for x in range(256):
             arrays.append({str(random.random() * 20): int(random.random() * 1000000)})
         test_object[str(random.random() * 20)] = arrays
-    run_encode(COUNT)
+    run_encode(COUNT, libraries)
 
     decode_data = json.dumps(test_object)
-    run_decode(COUNT)
+    run_decode(COUNT, libraries)
 
     decode_data = None
 
     results_new_benchmark(
         "Dict with 256 arrays with 256 dict{string, int} pairs, outputting sorted keys"
     )
-    run_encode_sort_keys(COUNT)
+    run_encode_sort_keys(COUNT, libraries)
 
     test_object = None
 
 
-def benchmark_complex_object():
+def benchmark_complex_object(libraries, factor=1):
     global decode_data, test_object
     results_new_benchmark("Complex object")
-    COUNT = 100
+    COUNT = int(100 * factor)
 
     with open(os.path.join(os.path.dirname(__file__), "sample.json")) as f:
         test_object = json.load(f)
-    run_encode(COUNT)
+    run_encode(COUNT, libraries)
 
     decode_data = json.dumps(test_object)
     test_object = None
-    run_decode(COUNT)
+    run_decode(COUNT, libraries)
 
     decode_data = None
 
 
 # =============================================================================
 # Main.
 # =============================================================================
+
+
+def main():
+    import argparse
+    import importlib
+
+    parser = argparse.ArgumentParser(
+        prog="ujson-benchmarks",
+        description="Benchmark ujson against other json implementations",
+    )
+
+    known_libraries = [
+        "ujson",
+        "nujson",
+        "orjson",
+        "simplejson",
+        "json",
+    ]
+
+    parser.add_argument(
+        "--disable",
+        nargs="+",
+        choices=known_libraries,
+        help=("Remove specified libraries from the benchmarks"),
+    )
+
+    parser.add_argument(
+        "--factor",
+        type=float,
+        default=1.0,
+        help=("Specify as a fraction speed up benchmarks for development / testing"),
+    )
+
+    parser.add_argument(
+        "command",
+        nargs="?",
+        help=(
+            "Exists for backwards compatibility. "
+            'Can be "skip-lib-comps" to disable computing benchmarks '
+            "for other json libraries."
+        ),
+        default=None,
+    )
```

Comment:
We really do not need to keep this script backwards compatible.

Reply:
Removed.